plugin.video.catchuptvandmore/addon.xml

```diff
@@ -1,32 +1,42 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="plugin.video.catchuptvandmore"
        name="Catch-up TV &amp; More"
-       version="0.1.0"
+       version="0.1.2"
        provider-name="SylvainCecchetto">
   <requires>
-    <import addon="xbmc.python" version="2.25.0"/>
+    <import addon="xbmc.python" version="2.24.0"/>
```

Comment:
You are targetting Krypton so you don't need simplejson. The python api version was correct as well.